neurom/viewer/view.py

```diff
@@ -37,7 +37,7 @@
 from neurom.core._soma import SomaCylinders
```

Comment:
my only nit is whether this should be called view anymore; part of the reason for this refactor is because .viewer and .vew are a strange naming, but having viewer.view also seems weird to me

I wonder if it should be 'plot.py', and then the functions can be renamed from plot_tree -> tree.  I'm not convinced of this, though, so it's more of a request for discussion

Reply:
I agree that viewer.view is weird and confusing. However, I would avoid to use plot because usually the convention is to use plot for functions, histograms, etc and visualize (or view, draw) for an object such as a neuron, tree, etc.

Reply:
Indeed view.py is not explicit. First thing, if we keep this file, regardless of the name, I think it should be preceeded by a leading underscore to make it clear that there are no public functions here.

Maybe we could restructure the following way: plot_dendrogram function goes into _dendrogram, 2D stuff goes to _view_2d.py and 3d stuff goes to _view_3d.py. The content of the folder would then be:

- __init__.py
- common.py (eventually to be renamed into _common.py)
- _view_2d.py
- _view_3d.py
- _dendrogram.py
- plotly.py (no underscore because it contains the public function `draw`)